HeartbeatLogging/Tests/Integration/HeartbeatLoggingIntegrationTests.swift

```diff
@@ -13,16 +13,307 @@
 // limitations under the License.
 
 import XCTest
-import HeartbeatLogging
+@testable import HeartbeatLogging
+
+private enum Constants {
+  static let heartbeatsDirectoryPath = "google-heartbeat-storage"
+  static let heartbeatsSuiteName = "com.google.heartbeat.storage"
```

Comment:
> at least one test fails when these values get out of sync with the other private enum, right?

Correct, this will catch hardcoded storage location regressions. 

Reworked in 66719b75d